fiftyone/utils/cvat.py

```diff
@@ -124,6 +125,9 @@ def import_annotations(
         group_id_attr (None): an optional attribute name in which to store the
             group id for labels
         backend ("cvat"): the name of the CVAT backend to use
+        anno_key (None): an optional annotation key to use to store the
```

Comment:
The fact that this method uses an annotation run behind the scenes to import the annotations is not intended to be leaked to the user. It's just a convenient way to load the annotations from CVAT.

Reply:
I put the arg so that a user can specify what key to use... 
If you call this method to import existing annotations, how would you later access the results?

Reply:
The annotations are placed onto your dataset. No need to access the `results` object. If you want directional workflows between a CVAT task and FO; you need to use `annotate()` + `load_annotations()`.

Reply:
> So the goal here is to gracefully continue if task 51/100 fails? That's fwm, but I believe this PR should be rebranded as an enhancement because there's not a "bug", it's just a different behavior.

No, the bug is that imported annotations are getting downloaded and saved with a key like `tmp_` and then automatically getting deleted regardless of success. And when multiple tasks are downloaded, the results of each will overwrite the others. 

But perhaps I don't really understand what an annotation_run's results contain...

Reply:
`delete_annotation_run()` doesn't delete the labels from the dataset. It just deletes the run metadata from the `runs` collection